libs/core/src/components/sage-input/sage-input.scss

```diff
@@ -0,0 +1,111 @@
+:host {
+  /**
+
+    * @prop --background: Background of the input
```

Comment:
Absolutely love listing all the props like this here

Reply:
+1 lets do this all the time 🔥